quick-integration.md

```diff
@@ -19,6 +20,17 @@ Once you finish with the integration we will have a call to check that your impl
 * Check the status of transactions both via webhooks and manually
 * Handling and cancelling failed transactions
 
+## Software Development Kits
+
+To facilitate easier integration with our API, we have SDKs available for the following languages:
+
+- [Java 7](sdks/java7.md)
+- [Java 8](sdks/java8.md)
+- [Ruby](sdks/ruby.md)
+- [Javascript](sdks/javascript.md)
+- [PHP](sdks/php.md)
+- [.NET](sdks/dotnet.md)
+
 Please read our [README](README.md) and the [Transaction Flow Guide](transaction-flow.md) before going through the implementation, but to make the process of creating a minimum approved integration easier, we propose you follow the following steps in this order:
 
 ## Authentication
```

Comment:
We could re-word this section to try to push people into using the SDKs - especially as we know that for example the Java example we published doesn't work properly (for PATCH endpoints)